lib/uplink/example_test.go

```diff
@@ -0,0 +1,243 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"storj.io/storj/lib/uplink"
+	"storj.io/storj/pkg/storj"
+)
+
+const (
```

Comment:
Set these inside the example rather than outside.

Reply:
@aligeti what with this comment?

Reply:
done